docs/admin/install.rst

```diff
@@ -0,0 +1,243 @@
+
+  .. code-block:: none
+
+    [securedrop-workstation-temporary]
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation
+    enabled=1
+    baseurl=https://yum.securedrop.org/workstation/dom0/f25
+    name=SecureDrop Workstation Qubes initial install bootstrap
+
+- Download the SecureDrop Workstation config package with the command:
+
+  .. code-block:: sh
+
+    dnf download securedrop-workstation-dom0-config
+
+- In the ``dom0`` administration terminal, run the following command to transfer the RPM package to dom0 and install SecureDrop Workstation:
+
+  .. code-block:: sh
+
+    qvm-run --pass-io work 'cat /home/user/securedrop-workstation-dom0-config-<versionNumber>-1.fc25.noarch.rpm' > securedrop-workstation.rpm
+    sudo dnf install securedrop-workstation.rpm
+
+  where ``<versionNumber>`` is the current SecureDrop Workstation release version. When prompted, press **Y** to install the package.
+ 
+Configure SecureDrop Workstation
+--------------------------------
+Before setting up the set of VMs used by SecureDrop Workstation, you must configure the Journalist Interface connection and submission key. 
+
+- To add the submission key, run the following command in the ``dom0`` terminal:
+
+  .. code-block:: sh
+
+    sudo cp /tmp/sd-journalist.sec /usr/share/securedrop-workstation-dom0-config/
+
+- Next, create the SecureDrop Workstation configuration file:
+
+  .. code-block:: sh
+   
+    cd /usr/share/securedrop-workstation-dom0-config
+    sudo cp config.json.example config.json
+
+- The ``config.json`` file must be updated with the correct values for your instance. Open it with root privileges in a text editor and update the following fields' values:
+
+  - **submission_key_fpr**: use the value of the submission key fingerprint
+  - **hidserv.hostname**: use the hostname of the Journalist Interface, including the ``.onion`` TLD
+  - **hidserv.key**: use the value of the v2 HidServAuth token for the Journalist Interface, or the v3 private authorization key value if your SecureDrop instance uses v3 onion services
+  - **environment**: use the value ``prod``
+
+You can find the values for the **hidserv.*** fields in the ``/tmp/journalist.txt`` file that you created in ``dom0`` earlier.
+
+- Finally, in the ``dom0`` terminal, run the command:
+
+  .. code-block:: sh
+
+    securedrop-admin --apply
+
+This command will take a considerable amount of time, as it sets up multiple VMs and installs supporting packages. When it completes, your SecureDrop Workstation is finally ready to use!
+
+Test the Workstation
+--------------------
+
+To start the SecureDrop Workstation client, double-click the SecureDrop desktop icon that was set up by the previous command. The client launcher will start and check for updates. The system should be up-to-date and no updates should be required, but if updates are available follow the instructions in the launcher to apply them.
```

Comment:
launcher -> preflight updater (launcher is an internal name)

Reply:
:heavy_check_mark: